content/docs/design/responsive_amp.md

```diff
@@ -22,7 +22,7 @@ It is super easy to make elements responsive in AMP. Just put `layout="responsiv
 
 ## Add styles to a page
 
-Add all CSS inside a `<style amp-custom>` tag in the head of the document.
+Migrate CSS inside a `<style amp-custom>` tag in the head of the document.
```

Comment:
The word "migrate" assumes they are converting a page, which is always the case.  How about jus "Add your"?

Reply:
Ah, good point - done.